app/boards/arm/adv360pro/adv360pro.keymap

```diff
@@ -0,0 +1,52 @@
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+#include <dt-bindings/zmk/rgb.h>
+#include <dt-bindings/zmk/backlight.h>
+
+/ {
+    behaviors {
+        #include "macros.dtsi"
```

Comment:
I just realized that this will be an issue with the setup script, since it will copy the keymap+conf and not any other files in this folder when setting up a user config repo. So this file will not exist and user builds will fail.

Hence I'd recommend embedding this dtsi here and removing the file. Or better yet remove it altogether, since they aren't used in the default keymap and docs cover usage examples.

Reply:
Removed it altogether now